stdlib/public/core/CTypes.swift

```diff
@@ -219,6 +219,29 @@ extension UInt {
 }
 
 /// A wrapper around a C `va_list` pointer.
+#if arch(arm64) && os(Linux)
```

Comment:
We should really just  add a new `_va_list_kind` `#if` condition.  This is not nearly as complicated as you might think, and it doesn't need to be done elegantly because it will never, ever need to be made part of the documented language.

Just add a new enumerator to `PlatformConditionKind`, add a new case to `getPlatformConditionKind` in `ParseIfConfig.cpp`, and add a new case to `LangOptions::checkPlatformConditionSupported` which matches some strings up with the appropriate cases of `getBultinVaListKind()`.